config/cli.go

```diff
@@ -79,6 +80,11 @@ func (cli *Cli) ShouldMapic() bool {
 	return cli.APIServer != ""
 }
 
+// Should we attempt communication with Mist?
+func (cli *Cli) ShouldMist() bool {
```

Comment:
What happens with Catalyst if not? Does running catalyst without Mist makes any sense?

Reply:
I think lots of folks hacking on catalyst-api boot it up without Mist, but the default case should definitely be to crash if there's no Mist there, that indicates a problem with the node. @thomshutt's suggestion was to just have a `-no-mist` parameter for that case, so I'll set that up.